doc/api/fs.md

```diff
@@ -1180,8 +1180,9 @@ to be notified of filesystem changes.
 
 If the underlying functionality is not available for some reason, then
 `fs.watch` will not be able to function.  For example, watching files or
-directories on network file systems (NFS, SMB, etc.) often doesn't work
-reliably or at all.
+directories on network file systems (NFS, SMB, etc.) and on exposed host file
+systems when using virtualization software (Vagrant, Docker, etc.) often doesn't
+work reliably or at all.
 
```

Comment:
minor nit... I'd restructure the sentence just a bit...

Something like:
```
For example, watching files or directories can be unreliable, and in some cases impossible, on network file systems (NFS, SMB, etc), and on host file systems when using virtualization software such as Vagrant, Docker, etc.
```
But I'm good with this also ;-)